doc/languages-frameworks/qt.section.md

```diff
@@ -0,0 +1,113 @@
+
+Qt
+=
+This section describes the differences between Nix expressions for Qt libraries and applications and Nix expressions for other C++ software. Some knowledge of the latter is assumed.
+
+There are primarily two problems which the Qt infrastructure is designed to address: ensuring consistent versioning of all dependencies and finding dependencies at runtime.
+
+**Nix expression for a Qt package (default.nix)**
+
+```nix
+{ mkDerivation, lib, qtbase }: #1
+mkDerivation { #2
+  pname = "myapp";
+  version = "1.0";
+
+  buildInputs = [ qtbase ]; #3
+}
+```
+
+* **#1:** Import `mkDerivation` and Qt (such as `qtbase` modules directly. *Do not* import Qt package sets; the Qt versions of dependencies may not be coherent, causing build and runtime failures.
+
+* **#2:** Use `mkDerivation` instead of `stdenv.mkDerivation`. `mkDerivation` is a wrapper around `stdenv.mkDerivation` which applies some Qt-specific settings. This deriver accepts the same arguments as `stdenv.mkDerivation`; refer to [Chapter 6, The Standard Environment](#chap-stdenv), The Standard Environment for details.
+
+> To use another deriver instead of `stdenv.mkDerivation`, use `mkDerivationWith`:
+
+```nix
+mkDerivationWith myDeriver {
+  # ...
+}
+```
+
+> If you cannot use `mkDerivationWith`, please refer to Locating runtime dependencies below.
```

Comment:
This does not look right, it should be multi-paragraph list item and I have no idea where are the blockquotes coming from.

Reply:
Have made the changes. thanks :)